website/content/en/preview/faq.md

```diff
@@ -133,3 +133,24 @@ See [Deprovisioning nodes]({{< ref "./tasks/deprovisioning" >}}) for information
 ### How do I upgrade Karpenter?
 Karpenter is a controller that runs in your cluster, but it is not tied to a specific Kubernetes version, as the Cluster Autoscaler is.
 Use your existing upgrade mechanisms to upgrade your core add-ons in Kubernetes and keep Karpenter up to date on bug fixes and new features.
+
+### How to upgrade from previous version (before v0.6.1) to v0.6.2
+Karpenter v0.6.2 requires a new permission `ec2:DeleteLaunchTemplate` to clean up Launch Templates.
+The IAM Role used for IRSA is now named something predictable rather than eksctl's long IDs.
+To upgrade, please follow the following steps.
+```
+helm uninstall karpenter --namespace karpenter
+
+TEMPOUT=$(mktemp)
+curl -fsSL https://karpenter.sh/preview/getting-started/cloudformation.yaml  > $TEMPOUT \
+&& aws cloudformation deploy \
+  --stack-name "Karpenter-${CLUSTER_NAME}" \
+  --template-file "${TEMPOUT}" \
+  --capabilities CAPABILITY_NAMED_IAM \
+  --parameter-overrides "ClusterName=${CLUSTER_NAME}"
+
+PHYSICAL_RESOURCE_ID="$(aws cloudformation describe-stack-resources --stack-name eksctl-${CLUSTER_NAME}-addon-iamserviceaccount-karpenter-karpenter --query "StackResources[0].PhysicalResourceId" --output text)"
+export KARPENTER_IAM_ROLE_ARN="$(aws iam get-role --role-name $PHYSICAL_RESOURCE_ID --query Role.Arn --output text)"
+```
+Then you can continue to install Karpenter by following [Install Karpenter Helm Chart](..//getting-started/#install-karpenter-helm-chart)
```

Comment:
I'm also a bit concerned that most users aren't actually using this guide, and instead using terraform. It would be great to have generic release instructions like I posted above. We may not be able to provide the explicit commands, but users can go to the getting started guide that they used and rerun the IAM setup. We could potentially link to those sections?

Reply:
OK, Shall we say those commands are example? It can be useful for the commands that return the IAM role Arn. It can be very confusing with generic instructions because of the messy names eksctl assigned.

Reply:
done